tools/file_packager.py

```diff
@@ -155,7 +155,7 @@ def has_hidden_attribute(filepath):
 
   try:
     attrs = ctypes.windll.kernel32.GetFileAttributesW(
-        u'%s' % filepath)
+        '%s' % filepath)
```

Comment:
In this case I don't think we need the % at all.  Just `ctypes.windll.kernel32.GetFileAttributesW(filepath)` should be enough these days I think.